id.ftd

```diff
@@ -205,3 +209,5 @@ id available for element which you want to refer to.
 
 -- ft.image:
 src: $assets.files.images.ftd-give-me-an-element-with-id.png
+scale: 1
```

Comment:
Why `scale: 1` ?

Reply:
after giving scale: 1, the image size issues have solved.